modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/CreateAnAPIApplicationEndpoint.testcase

```diff
@@ -58,23 +61,42 @@ definition {
 		}
 	}
 
-	@priority = 5
-	test CanCreateEndpointWithSiteScope {
-		property portal.acceptance = "true";
+	@priority = 3
+	test CanCreateEndpointWithPathAndDescriptionWithSuccessAlert {
+		task ("And Given I go to 'Endpoints' tab in 'Edit API Application'") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Endpoints",
+				title = "test2");
+		}
```

Comment:
Could you check all the steps for this test? Thanks! there are some steps missing.